### ISSUE TITLE
[Android] - RemoteConfig is not working

### ISSUE THREAD

**Issue by zakharchenkoAndrii** [Countly/countly-sdk-react-native-bridge#238]
Hi there 👋 

After updating `countly-sdk-react-native-bridge` to (latest) `23.6.0` remoteConfig fetching stopped to work for Android _only_.

(server version is "[23.06.13](https://github.com/Countly/countly-server/blob/master/CHANGELOG.md#version-230613)")

After getting the key either by 

```
Countly.getRemoteConfigValueForKey("KeyName", function(data){
console.log(data);
});
```
or
```
var data = await Countly.getRemoteConfigValueForKeyP("KeyName");
``` 

I am receiving the value that looks like internal access _key_ for the value: `ly.count.android.sdk.RCData@<someGeneratedId>`

**Comment by peterBrxwn** [comment]
Hello, thank you for reporting this.

After looking into it, we have discovered the bug that causes "getRemoteConfigValueForKey" and "getRemoteConfigValueForKeyP" methods to return an object.

This bug has been fixed, and a new version (23.6.1) has been released.

**Comment by zakharchenkoAndrii** [comment]
Thank you, now it's working 🙁
